choreographer/src/main/java/eu/arrowhead/core/choreographer/ChoreographerController.java

```diff
@@ -220,6 +264,10 @@ public void removeActionPlanById(@PathVariable(value = PATH_VARIABLE_ID) final l
     @ResponseBody public void startPlan(@RequestBody final List<ChoreographerRunPlanRequestDTO> requests) {
```

Comment:
this is wrong:

1) 2 loops. First check preconditions, and if everything is fine, start plans in second loop.
2) Every plan is atomic. If a plan check is failed, we store the error, then proceed to the next plan. After the loop send the errors and successes back to the caller.